features/security_status.feature

```diff
@@ -131,15 +131,19 @@ Feature: Security status command behavior
             pro security-status --help
         for a list of available options\.
 
+        Make sure to run
```

Comment:
as discussed, please make this dynamic
a) not ran update ever
b) not ran update for x hours
c) do not show it at all if run <y hours

Reply:
@cpaelzer Done!
With the slight difference that it says days instead of hours - purely easiness of implementation;
If can't determine last time, just say it's outdated.
If > 1 day, tell how many days
otherwise just don't show.

Reply:
Thanks, good now!